test/Nancy.Tests/project.json

```diff
@@ -1,6 +1,8 @@
 {
+    "version": "1.4.1.0",

```

Comment:
Don't forget to look at this again. Instead of changing the version of this assembly, we could change the test data to the values @thomaslevesque suggested in https://github.com/NancyFx/Nancy/pull/2634#issuecomment-263126390.

Of course, either change would only fix the _test_. There's a larger issue of whether `DefaultObjectSerializer` should able to deserialize types into editions of those types from different-versioned assemblies, but that's beyond this PR. I could spin off an issue, if you like.